schemas/applicants/definitions.yaml

```
@@ -83,3 +85,22 @@ applicant_last_name:
   type: string
   pattern: ^[^!#$%*=<>;{}"]*$
   description: The applicant's surname
+
+applicant_consent_item:
+  type: object
+  required:
+    - name
+    - granted
+  properties:
+    name:
+      $ref: "#/applicant_consent_name"
+    granted:
+      type: boolean
+  description: The applicant's consents
```

Typo, should probably be 
```suggestion
  description: The applicant's consent
```